### PR TITLE
Add anyclick event

### PR DESCRIPTION
**Pull request by Pessimistress**
Add a new event `anyclick` (see documentation in this PR).
The naming is awkward and I plan to rename the current `click` to `singleclick` in the next major release.

### CODE REVIEW

File: examples/main/constants.js [hunk 1]
```diff
@@ -59,10 +60,6 @@ export const EVENTS = [
 export const INITIAL_OPTIONS = {
   click: true,
   doubletap: true,
-  pointermove: true,
```
Comment (ericsoco) [inline examples/main/constants.js]: Why are these removed?

Comment (Pessimistress) [reply examples/main/constants.js]: These are just default checked boxes in the example. I find them flood the log quite a bit.
